examples/gnrc_networking/Makefile

```diff
@@ -34,9 +34,19 @@ USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 USEMODULE += netstats_rpl
 
-# Set a custom 802.15.4 channel if needed
-DEFAULT_CHANNEL ?= 26
-CFLAGS += -DDEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
+# Set a custom channel if needed
+ifeq (,$(filter cc110x,$(USEMODULE)))     # radio is cc110x sub-GHz
```

Comment:
Well, after trying again, this doesn't work actually. `$(USEMODULE)` only contains the modules defined above in this particular Makefile. The modules dependencies is processed __after__ so '$(USEMODULE)' never contains `cc110x` and the code always end up in this block, even if I try to compile for a samr21-xpro (which uses an `at86rf2xx` driver).

I'm not an expert in Makefile so I may have missed something.